src/cmd/is_ready.go

```diff
@@ -0,0 +1,92 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/f1bonacc1/process-compose/src/client"
+	"github.com/f1bonacc1/process-compose/src/types"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+	"strings"
+	"time"
+)
+
+var (
+	waitReady = false
```

Comment:
Can we move this to `src/config/Flags.go`?

Reply:
Done!